src/Nowruz/modules/userProfile/containers/verifyEducationModal/index.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import variables from 'src/components/_exports.module.scss';
+import { Icon } from 'src/Nowruz/general/Icon';
+import { AccountItem } from 'src/Nowruz/modules/general/components/avatarDropDown/avatarDropDown.types';
+import { AvatarLabelGroup } from 'src/Nowruz/modules/general/components/avatarLabelGroup';
+import { Button } from 'src/Nowruz/modules/general/components/Button';
+import { Checkbox } from 'src/Nowruz/modules/general/components/checkbox/checkbox';
+import { FeaturedIcon } from 'src/Nowruz/modules/general/components/featuredIcon-new';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/Nowruz/modules/general/components/modal';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+
+import { useVerifyExperienceModal } from './useVerifyEducationModal';
+import { OptionType, VerifyEducationModalProps } from './verifyEducationModal.types';
+
+export const VerifyEducationModal: React.FC<VerifyEducationModalProps> = ({
+  open,
+  handleClose,
+  education,
+  organization,
+  onSendRequest,
+}) => {
+  const {
+    register,
+    errors,
+    days,
+    months,
+    years,
+    day,
+    month,
+    year,
+    forgotInfo,
+    handleSubmit,
+    onSend,
+    onSelectDay,
+    onSelectMonth,
+    onSelectYear,
+    handleForgotInfo,
+  } = useVerifyExperienceModal(education, organization, handleClose, onSendRequest);
```

Comment:
`useVerifyEducationModal` is more relevant.
this is not important again but help us in searching the whole project